config.go

```diff
@@ -76,6 +77,8 @@ type Config struct {
 	// TODO(roasbeef): use the Taproot Asset chain param wrapper here?
 	ChainParams chaincfg.Params
 
+	Lnd *lndclient.LndServices
```

Comment:
This needs to be set somewhere, otherwise we'll produce a nil pointer panic.
This makes me think we should probably also do a quick call to this new RPC in one of the existing integration tests.

Reply:
Thanks for the review!

In the latest commits: I've initialised `Lnd` and I've added an itest to ensure that the GetInfo RPC returns without error.